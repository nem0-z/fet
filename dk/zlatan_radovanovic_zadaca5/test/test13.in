print foo_ + 5, "bar = " & bar1, 10.0
