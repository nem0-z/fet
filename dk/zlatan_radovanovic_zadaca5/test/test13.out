5.000000
bar = 0.000000
10.000000
